phpstan.neon

```diff
@@ -0,0 +1,12 @@
+parameters:
+    level: max
+    paths:
+        - src
+        - tests
+    autoload_directories:
+    	- tests
+    ignoreErrors:
+        # Ignore some errors triggered by testing.
+        - '#Access to an undefined property class@anonymous/tests/Request/BaseRequestTest#'
+        - '#Access to an undefined property DDB\\OpenPlatform\\Response\\Response::\$unknown#'
+        - '#Access to an undefined property DDB\\OpenPlatform\\Request\\GenericRequest::\$aProperty#'
```

Comment:
Could we use a more generic string and then [scope the ignore to everything under `tests`](https://github.com/phpstan/phpstan#ignore-error-messages-with-regular-expressions)?.